### ISSUE TITLE
Add an option to change FileSystem dock behavior for auto scrolling when opening a scene.

### ISSUE THREAD

**Issue by avencherus** [godotengine/godot-proposals#1705]
**Describe the project you are working on:**
2D Games

**Describe the problem or limitation you are having in your project:**
The problem is when opening multiple scenes in sequence.  The scroll point jumps to place the last opened scene at the bottom.  It's not desirable, as when opening many scenes in order it requires scrolling again.  It can be worked around by some degree if you know the behavior and work from the bottom up.

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/13004169/96898575-8bebc100-1498-11eb-9a73-f26c00e866c4.gif)

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
I wasn't able to find any options to control this, so it would be great if the editor settings had an option to disable this automatic realignment.

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**

```
if (file_dock_scroll_auto_align) {
	
	file_dock.scroll_to(x)
}
// else don't do it.
```

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
No.

**Is there a reason why this should be core and not an add-on in the asset library?:**
Deals directly with editor behavior.  Unclear if hacky plugins could do the job.

**Comment by Calinou** [comment]
Maybe we should disable this behavior entirely?

**Comment by avencherus** [comment]
Yeah, was on the fence of posting a proposal or issue.  Wasn't sure of the design intentions behind it, or if it was unintentional.
